stdlib/format.mli

```diff
@@ -152,6 +152,9 @@ type formatter
 
 *)
 
+val pp_infinity : int
+(** [pp_infinity] is a large value that is used as default token size *)
```

Comment:
This is not what `pp_infinity` means for users of `Format`. We could move the item in the margin section and explains that this is the maximal size of the `margin`:
```
[pp_infinity] is the maximal size of the margin. Its exact value is implementation dependent but is guaranteed to be greater than 10^9.
```

Reply:
Done!